bot.py

```diff
@@ -48,7 +48,7 @@
 TOKEN = str(config.get("server", "token"))
 botname = str(config.get("server", "name"))
 prefix = str(config.get("server", "prefix"))
-language = str(config.get("server", "language"))
+language = str(config.get("server", "language")) if config.has_option("server", "language") else "en"
```

Comment:
Some check here to make sure the configured language is available

Reply:
This might look cleaner (untested)? But it's a style choice.
```suggestion
language = str(config.get("server", "language", fallback="en"))
```

Reply:
I like your style choice @d7415 - I was looking for something similar but didn't come across this.
@Edwinexd the check is in i18n.get(), along with missing translations for existing languages. KeyError should be raised for both bad languages and missing translations, meaning it will fallback to english.

I will make the check only for `rl!language`, if someone messes with the `config.ini` directly, I hope they know what they are doing. Even if they set language = abcd, everything should be sent in english. I will make sure the error is actually caught by KeyError during testing.